libs/media-player/src/lib/as-two-digit-string.ts

```diff
@@ -0,0 +1,9 @@
+import { isInteger } from '@coscrad/validation-constraints';
+
+export const asTwoDigitString = (inputNumber: number) => {
```

Comment:
Have we written this logic elsewhere? Maybe we need a lib for time?

Reply:
Probably, but on a separate PR?  I think there might be a similar but slightly different function on the frontend.  On a search I found `convertMillisecondsToSeconds`